ldap/servers/slapd/back-ldbm/db-mdb/mdb_instance.c

```diff
@@ -97,6 +97,9 @@ typedef struct {
 static dbmdb_dbi_t *dbi_slots;    /* The alloced slots */
 static int dbi_nbslots;           /* Number of available slots in dbi_slots */
 
+static int32_t g_mdb_env_is_open = false;
```

Comment:
Will it work correctly for multiple backends? (I might misunderstand how we handle multiple LMDB environments though. So, just making sure its thought through)

Reply:
There is a single environment (except for the temporary db used during import to handle entryrdn but that is another part of the code).
txn may need to handle different backends to keep consistency (moddn operation or some plugins like referntial interghrity plugin or memberof) so we need a single environment